addons/mail/static/src/models/message_action_view.js

```diff
@@ -0,0 +1,207 @@
+/** @odoo-module **/
+
+import { registerModel } from '@mail/model/model_core';
+import { attr, one } from '@mail/model/model_field';
+import { clear, insertAndReplace } from '@mail/model/model_field_command';
+import { markEventHandled } from '@mail/utils/utils';
+
+registerModel({
+    name: 'MessageActionView',
+    identifyingFields: ['messageAction'],
```

Comment:
Rebase master and adapt with new change with identifying field.
Here don't provide `identifyingMode` (will use default mode, which `"and"`), and put `identifying: true` on `messageAction` field.